components/steps/index.tsx

```diff
@@ -18,8 +18,15 @@ export interface StepsProps {
   style?: React.CSSProperties;
 }
 
+export interface StepProps {
+  description?: React.ReactNode;
+  icon?: React.ReactNode;
+  status?: string;
```

Comment:
Field status should be "wait", "process", "finish" or "error".
https://ant.design/components/steps/#Steps.Step

Reply:
I will fix it!